prebid_cache_client/client.go

```diff
@@ -40,6 +47,7 @@ type Cacheable struct {
 }
 
 func NewClient(conf *config.Cache) Client {
+	PrebidCacheURL = conf.GetBaseURL() + "/cache"
```

Comment:
Shouldn't we make the `path` configurable too? @guscarreon @hhhjort

Reply:
Implementing the configurable feature...

Reply:
Implementing the configurable feature..

Reply:
done